publishing-bot/Makefile

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+-include $(CONFIG)
+-include $(CONFIG)-token
+
+DOCKER_REPO ?= k8s-publishing-bot
+NAMESPACE ?=
+TOKEN ?=
+KUBECTL ?= kubectl
+SCHEDULE ?= 0 5 * * *
+INTERVAL ?= 86400
+MEMORY_REQUESTS ?= 200Mi
+MEMORY_LIMITS ?= 1.6Gi
+
+prepare_spec = sed 's,DOCKER_IMAGE,$(DOCKER_REPO),g;s,MEMORY_REQUESTS,$(MEMORY_REQUESTS),g;s,MEMORY_LIMITS,$(MEMORY_LIMITS),g'
+
+init-deploy:
+	$(KUBECTL) delete -n "$(NAMESPACE)" --ignore-not-found=true deployment publisher
```

Comment:
Can we keep this as ReplicaSet as in the publishing-bot repo (for simplicity)? Or else, we'll need to first delete the existing ones in the cluster and then recreate if we use the modified script...

https://github.com/kubernetes/publishing-bot/blob/da9508d9d841255b64fc7ace6dc6010c5e225a5b/Makefile#L69

Reply:
It's done.